src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Api.Enduser/Authorization/AuthorizationReguiremnent/EndUserResourceAccessRequirement.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Authorization;
+
+namespace Altinn.AccessManagement.Api.Enduser.Authorization.AuthorizationReguiremnent
+{
+    /// <summary>
+    /// Requirement for authorization policies used for accessing apis. https://docs.asp.net/en/latest/security/authorization/policies.html
+    /// for details about authorization in asp.net core.
+    /// </summary>
+    public class EndUserResourceAccessRequirement : IAuthorizationRequirement
+    {
+        /// <summary>
+        /// Gets or sets The Action type defined for the policy using this requirement
+        /// </summary>
+        public string ActionType { get; set; }
+
+        /// <summary>
+        /// Gets or sets the resourcId for the resource that authorization should verified for
+        /// </summary>
+        public string ResourceId { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the policy should allow access for unauthorized parties
+        /// </summary>
+        public bool AllowAllowUnauthorizedParty { get; set; }
```

Comment:
På sikt vil vi også trenge scope krav input her.
For nå kan vi evt legge til  hardkodet `altinn:portal/enduser`